customerio/api.py

```diff
@@ -0,0 +1,15 @@
+from .client_base import ClientBase
+
```

Comment:
Should add a one line comment at the top explaining what this is. Can also refer to transactional's API page.